src/App.tsx

```diff
@@ -43,15 +43,16 @@ function App() {
   return (
     <div className="m-[15%]">
       <div className="grid grid-cols-[repeat(auto-fill,minmax(200px,1fr))]">
-        <Clock
-          name="Quartz"
-          tz="America/Los_Angeles"
-          twitter="piezo_electric"
-        />
-        <Clock name="Ava" tz="America/Los_Angeles" twitter="avaaa_42" />
+        <Clock name="Ava" tz="America/New_York" twitter="avapsilver" />
```

Comment:
aren't you in Idaho?

Reply:
im gonna be there for literally 5 days and im still in CA rn 💀

Reply:
the whole point is to keep it updated :sob: